src/standard/id.ml

```diff
@@ -80,6 +80,7 @@ let tracked ~track ns path =
 (* ************************************************************************* *)
 
 let ac_symbol = mk Attr "ac"
+let predicate_decl = mk Attr "predicate"
```

Comment:
maybe `predicate_def` rather than `decl` ? Usually decl means declaration i.e. mainly stating the type of a symbol, whereas `def` means definition and thus associates a term/body to the symbol.

Reply:
Yes, I agree. It's done